PDO/PDO.php

```diff
@@ -1115,7 +1115,12 @@ public function query($statement, $mode = PDO::ATTR_DEFAULT_FETCH_MODE, $arg3 =
      * @see PDOStatement::setFetchMode For a full description of the second and following parameters.
      */
     #[PhpStormStubsElementAvailable('8.0')]
-    public function query($statement, $mode = PDO::ATTR_DEFAULT_FETCH_MODE, ...$fetch_mode_args) {}
+    #[LanguageLevelTypeAware(['8.0' => 'array'], default: 'PDOStatement|false')]
```

Comment:
Could you please clarify why the return type should be an array since 8.0? According to https://www.php.net/manual/en/pdo.query.php and https://github.com/php/php-src/blob/master/ext/pdo/pdo_dbh.stub.php#L422 return type is `PDOStatement|false`

Reply:
Thanks to your @isfedorov comment I was able finally to figure out how it should looks like